lib/main/atom-ide/datatipProvider.tsx

```diff
@@ -48,15 +48,25 @@ export class TSDatatipProvider implements DatatipProvider {
       const data = result.body!
       const code = await highlightCode(data.displayString.replace(/^\(.+?\)\s+/, ""))
       const kind = (
-        <div class="atom-typescript-datatip-tooltip-kind">
+        <div className="atom-typescript-datatip-tooltip-kind">
           {data.kind}
           {data.kindModifiers ? <i> ({data.kindModifiers})</i> : null}
         </div>
       )
-      const docs = <div class="atom-typescript-datatip-tooltip-doc">{data.documentation}</div>
+      const tags = data.tags.map(tag => (
+        <div className="atom-typescript-datatip-tooltip-doc-tag">
+          {tag.name} - {tag.text}
```

Comment:
Perhaps would be nice to emphasize the `tag.name` part with inline style, instead of generic `-`? Also, perhaps adding a class dependent on the tag name could be useful.

For instance:
```ts
const tagClass = `atom-typescript-datatip-tooltip-doc-tag ` +
  `atom-typescript-datatip-tooltip-doc-tag-name-${tag.name}`
<div class={tagClass}>
  <span class="atom-typescript-datatip-tooltip-doc-tag-name">{tag.name}</span>
  <span class="atom-typescript-datatip-tooltip-doc-tag-text">{tag.text}</span>
</div>
```

Also, while at it, when `tag.name === 'param'`, the first "word" (that is, the thing that matches `/^\s*(\w+)/`) in `tag.text` is the parameter name. Would be nice to extract and style it somehow I reckon. The "easy" option would be something like
```ts
const [firstWord, restOfText...] = tag.text.split(/\s+/g)
// snip
  <span class="atom-typescript-first-word">{firstWord}</span> {restOfText}
```
and then add css rules using `tag.name`-dependent class on the outer `div` (if that is implemented).

Reply:
Actually, my first implem was very close than what you propose here for the DOM structure, but I thought for a second that it was maybe overkill. Let's do this then !

I also love the idea of splitting the first world this way. I'll come up with something and let you know !